spec/requests/invites_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'helpers/rails_helper'
+require 'helpers/api_helper'
+
+RSpec.describe "Invites", type: :request do
+  include_context "with authenticated requests"
+  include_context "with JSON responses"
+
+  let(:student) { create(:student) }
+  let(:teacher) { create(:teacher) }
```

Comment:
The creation of the teacher inside the course factory is just there to provide a default, if we are using the teacher anywhere else in the spec, better to create it outside.